src/develop/imageop.c

```diff
@@ -1964,10 +1964,10 @@ GtkWidget *dt_iop_gui_get_expander(dt_iop_module_t *module)
   /* add empty space around module widget
    * this cannot be set in CSS because the module collapsing is badly handled
    * */
-  gtk_widget_set_margin_start(module->widget, DT_PIXEL_APPLY_DPI(4));
-  gtk_widget_set_margin_end(module->widget, DT_PIXEL_APPLY_DPI(4));
-  gtk_widget_set_margin_top(module->widget, DT_PIXEL_APPLY_DPI(4));
-  gtk_widget_set_margin_bottom(module->widget, DT_PIXEL_APPLY_DPI(4));
+  gtk_widget_set_margin_start(module->widget, DT_PIXEL_APPLY_DPI(16));
```

Comment:
I find this is loosing quite some space around the modules. I prefer to keep 4.

Reply:
the reason for this is it keeps the content aligned on the module title, and the optical margin it creates is clean. Also, I have removed the outer margin of the modules, so overall I think the width of the actual content does not change.

Reply:
On my side I have a noticable change!

Before:
![capture d ecran de 2019-02-13 19-55-44](https://user-images.githubusercontent.com/467069/52736323-e34e4400-2fc9-11e9-9b16-5369cd455cf6.png)

Now:
![capture d ecran de 2019-02-13 19-57-54](https://user-images.githubusercontent.com/467069/52736334-ea755200-2fc9-11e9-8848-7bc6438a608f.png)

Reply:
> the reason for this is it keeps the content aligned on the module title

I don't see that. This is the margin (inside) so keeping 4 will just makes the borders left and top of button [tout] smaller.

Reply:
See here:
![capture d ecran du 2019-02-13 14-28-21](https://user-images.githubusercontent.com/2779157/52738336-707aa400-2f9c-11e9-904f-c9efda375796.png)